components/Send/components/ScannerAddress.tsx

```diff
@@ -71,36 +69,26 @@ const ScannerAddress: React.FunctionComponent<ScannerAddressProps> = ({ updateTo
 
   const { colors } = useTheme();
   return (
-    <View style={{ width: '100%', height: '100%' }}>
+    <View style={{
```

Comment:
I was testing in IOS... and you need to change this `View` to `SafeAreaView` from react-native. In Android have no effect but in IOS is important.